lib/zold/upgrades.rb

```diff
@@ -2,6 +2,22 @@
 
 module Zold
   # Class to manage data upgrades (when zold itself upgrades).
+  #
+  # This class will write the version file to the zoldata directory.
```

Comment:
@botanicus what happened to the draft? This text doesn't explain the case of missing version file.

Reply:
@ledestin added.